server/routes/search/handlers/caSearch.ts

```diff
@@ -84,8 +91,19 @@ export default class CaSearch {
           licenceStatus,
         }
       }),
+      attentionNeededResults: attentionNeededResults.map(res => {
+        return {
+          ...res,
+          releaseDate: res.releaseDate ? format(parseCvlDate(res.releaseDate), 'dd MMM yyyy') : 'not found',
```

Comment:
there's a filter for this: `cvlDateToDateShort` would it make sense to just pass the date through and use that?